sbpy/photometry/core.py

```diff
@@ -26,6 +27,24 @@
 from ..units import hundred_nm
 
 
+def _process_ephem_input(eph, key=None):
+    if eph is None:
+        return None, None
+    if not isinstance(eph, (Ephem, dict, np.ndarray, Number, u.Quantity)):
+        raise ValueError('`~sbpy.data.Ephem`, `dict`, `numpy.ndarray`, `Number`, or `~astropy.units.Quantity expected, {0} received'.format(type(eph)))
+    if isinstance(eph, dict):
+        eph = Ephem(eph)
```

Comment:
Actually initialization directly with `dict` does work.  But do you advise we should always use `.from_dict` for increased API stability?  If so, I can change the code.

Reply:
Hm. Interesting. I guess it works because `sbpy.data.DataClass` takes `*kwargs` as input parameter and then interprets the dictionary correctly. Yeah, maybe change it to use `.from_dict` - just to be on the safe side... Thanks!

Reply:
Sure.